<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiChannelList</class>
 <widget class="QDialog" name="uiChannelList">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>590</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Channel list</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../scc.qrc">
    <normaloff>:/images/logo16x16.png</normaloff>:/images/logo16x16.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>210</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
        <widget class="QGroupBox" name="groupBox_search">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>201</width>
           <height>151</height>
          </rect>
         </property>
         <property name="title">
          <string>Search</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>180</width>
            <height>121</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_channel_name">
             <property name="styleSheet">
              <string notr="true">font-weight:bold;</string>
             </property>
             <property name="text">
              <string>Channel name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_search"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="pushButton_search">
               <property name="text">
                <string>Search</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_clear">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_hide_empty_channels">
             <property name="text">
              <string>Hide empty channels</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_show_adv_options">
             <property name="text">
              <string>Show advanced</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_type">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>160</y>
           <width>201</width>
           <height>171</height>
          </rect>
         </property>
         <property name="title">
          <string>Type</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>22</y>
            <width>181</width>
            <height>136</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="checkBox_teen">
             <property name="text">
              <string>Teen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_common">
             <property name="text">
              <string>Common</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_erotic">
             <property name="text">
              <string>Erotic</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_thematic">
             <property name="text">
              <string>Thematic</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_regional">
             <property name="text">
              <string>Regional</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_category">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>340</y>
           <width>201</width>
           <height>191</height>
          </rect>
         </property>
         <property name="title">
          <string>Category</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>181</width>
            <height>164</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="checkBox_wild">
             <property name="text">
              <string>Wild</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_tame">
             <property name="text">
              <string>Tame</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_with_class">
             <property name="text">
              <string>Classy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_cult">
             <property name="text">
              <string>Cult</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_moderated">
             <property name="text">
              <string>Moderated</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_recommended">
             <property name="text">
              <string>Recommended</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_1">
         <attribute name="title">
          <string>All</string>
         </attribute>
         <widget class="QTableWidget" name="tableWidget_all">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>591</width>
            <height>511</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Channel name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Number of persons</string>
           </property>
          </column>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Close|QDialogButtonBox::Retry</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../scc.qrc"/>
 </resources>
 <connections/>
</ui>
